docs/supported.md

```diff
@@ -606,7 +606,7 @@ For color definition, KaTeX color functions will accept the standard HTML [pred
 |$\mathit{Ab0}$ `\mathit{Ab0}`  |$\mathtt{Ab0}$ `\mathtt{Ab0}`  |$\mathcal{AB0}$ `\mathcal{AB0}`
 |$\textit{Ab0}$ `\textit{Ab0}`  |$\texttt{Ab0}$ `\texttt{Ab0}`  |$\cal AB0$ `\cal AB0`
 |$\it Ab0$ `\it Ab0`            |$\tt Ab0$ `\tt Ab0`            |$\mathscr{AB}$ `\mathscr{AB}`
-|$\emph{Ab0}$ `\emph{Ab0}`      |                               ||
+|$\emph{Ab0}$ `\emph{Ab0}`      |$\mathsfit{Ab0}$ `\mathsfit{Ab0}`||
```

Comment:
I think I'd rather see this in the third column just below `\sf`. (Sorry maintaining these tables in text is a pain.)

This example doesn't work on the Netlify preview, but I think that's because it's using the released CSS instead of the branch CSS.